### ISSUE TITLE
Warning on First Use of Export-Configuration

### ISSUE THREAD

**Issue by aproposts** [PoshCode/Configuration#48]
The first time Export-Configuration is used, it generates a warning message:

`WARNING: The Enterprise path C:\Users\<username>\AppData\Roaming\powershell cannot be found`

...and it cannot be suppressed with any of the usual methods:

    $callerWarningPreference = $WarningPreference
    $WarningPreference = 'SilentlyContinue'
    
    $new | Export-Configuration -WarningAction Ignore
    
    $WarningPreference = $callerWarningPreference

I'm trying to present a clean experience to users and this error message is a bit cryptic and not at all helpful: Export-Configuration creates the path and writes the file without issue. Surely this message is better suited to the Verbose stream? Searching the source, it appears to be coming from [Get-ConfigurationPath](https://github.com/PoshCode/Configuration/blob/b9c46d552371ad86bc9b896f92f8a7c88051fb11/Source/Public/Get-ConfigurationPath.ps1#L83) and the flags to suppress warning aren't being pushed down into its scope.

I suppose I could call Get-ConfigurationPath myself first--suppressing the warning there--but I think the issue still deserves attention in either the code or the documentation.

**Comment by Jaykul** [comment]
Yeah, I think I can take a look at that. It's likely the extra locations are basically going unused anyway ;-)
